LICENCE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2023 anmode
```

Comment:
I do not think it is correct! please see other repo or example how to create a license...and this is open sourced startup project. So you have to think this is mind. thanks

Reply:
yup now I understand I'll send new PR, thank you!

Reply:
You do not need to create a New pR!

Reply:
you can continue in this one! too